cupsfilters/libcups2.c

```diff
@@ -0,0 +1,267 @@
+//
+// Image library test program for libcupsfilters.
+//
+// Copyright 2007-2011 by Apple Inc.
+// Copyright 1993-2006 by Easy Software Products.
+//
+// Licensed under Apache License v2.0.  See the file "LICENSE" for more
+// information.
+//
+// Contents:
+//
+//   main() - Main entry...
```

Comment:
Looks like that you have copied this header from another file. Please update it, especially line 2 and line12.